src/Models/BaseElement.php

```diff
@@ -822,4 +830,28 @@ public function getPageTitle()
 
         return null;
     }
+
+    /**
+     * @return boolean
+     */
+    public function isFirstElement()
+    {
+        return !($this->Parent()->Elements()->filter('Sort:LessThan', $this->Sort)->exists());
```

Comment:
What do you reckon about `$this->Parent()->Elements()->first() === $this`?

Reply:
Would `$this` be robust enough? I'm just thinking of objects perhaps changing at runtime with something like fluent etc.. 

Perhaps `$this->Parent()->Elements()->first()->ID === $this->ID`. ID is all that matters right?

Reply:
Yeah give that a go. Your tests should be good enough to try it out =D